### PR TITLE
metrics: compaction

### PR DESCRIPTION
**Pull request by zhangjinpeng1987**
ref https://github.com/pingcap/tikv/issues/1656
@siddontang @andelf PTAL

### CODE REVIEW

**Comment by siddontang** [comment]
PTAL @BusyJay @hhkbp2

File: src/raftstore/store/engine_metrics.rs [hunk 1]
```diff
@@ -37,7 +39,9 @@ pub const ENGINE_TICKER_TYPES: &'static [TickerType] = &[TickerType::BlockCacheM
                                                          TickerType::BytesWritten,
                                                          TickerType::BytesRead,
                                                          TickerType::IterBytesRead,
-                                                         TickerType::StallMicros];
+                                                         TickerType::StallMicros,
+                                                         TickerType::CompactReadBytes,
+                                                         TickerType::CompactWriteBytes];
```
Comment (BusyJay) [inline src/raftstore/store/engine_metrics.rs]: I prefer the following style:

```
[
    line1,
    line2,
]
```

**Comment by hhkbp2** [comment]
LGTM

File: src/raftstore/store/engine_metrics.rs (hunk 1 above)
Comment (siddontang) [inline src/raftstore/store/engine_metrics.rs]: Any update? @zhangjinpeng1987

Comment (zhangjinpeng1987) [inline src/raftstore/store/engine_metrics.rs]: this is formatted by rustfmt 0.6 automatically.

**Comment by siddontang** [comment]
CI failed 

```
please make format before creating a pr!!!
```

**Pull request merged**